### PR TITLE
Changes to make it work in Symfony 4

### PR DESCRIPTION
**Pull request by amorebietakoUdala**
- In order work in Symfony 4, I have changed the deprecated DefinitionDecorator for ChildDefinition as the migration docs say. It works without problem.

### CODE REVIEW

**Comment by Maks3w** [comment]
I see. Could you please rebase the branch?

Then I'will release a new major version with a minimum requirement of SF 3.4.

**Comment by amorebietakoUdala** [comment]
> 
> 
> I see. Could you please rebase the branch?
> 
> Then I'will release a new major version with a minimum requirement of SF 3.4.

I made the changes in the development branch. And make another pull request in that branch. It's that ok?

File: Driver/ZendLdapDriver.php [hunk 1]
```diff
@@ -86,6 +86,9 @@ public function bind(UserInterface $user, string $password): bool
 
     /**
      * Treat a Zend Ldap Exception.
+     *
```
Comment (Maks3w) [inline Driver/ZendLdapDriver.php]: Please undo the changes on this file and then it's ok.

**Comment by Maks3w** [comment]
Thanks. I've close `develop` branch as it's very outdated does not match with the current maintenance activity. Thanks any way

File: Driver/ZendLdapDriver.php (hunk 1 above)
Comment (amorebietakoUdala) [reply Driver/ZendLdapDriver.php]: Sorry I was on hollidays. I haven't read it until today. I'll try do what you say.

Comment (amorebietakoUdala) [reply Driver/ZendLdapDriver.php]: I think it's done.
